CHANGELOG.md

```diff
@@ -119,5 +119,110 @@ This option is new for VectorCAST 23 sp2
 - Added support for relative paths to the coded test file in test script #106
 - Fixed issue with unrecoverable link error after a coded test compile error #128
 
+## [1.0.11] - 2024-06-24
 
+### Code Maintenance
+- Increased timeouts for E2E tests.
+
+## [1.0.11] - 2024-06-27
+
+### Code Maintenance
+- Split E2E tests and isolated deletion tests.
+- Implemented dedicated generation and deletion tests for ATG (unit and function) and BASIS PATH (unit and function) test cases.
+- Ran Prettier and xo linter on typescript files.
+
+### Bug Fixes
+- Removed another instance where the VCAST version mismatch error was displayed in the output pane.
+
+## [1.0.11] - 2024-07-02
+
+### Code Maintenance
+- Divided E2E test specs into independent groups, allowing GitHub Actions to retrieve and run multiple vcastVersion-group combinations in parallel.
+
+## [1.0.11] - 2024-07-09
+
+### Code Maintenance
+- Updated the Docker test image to use Ubuntu 24.04 as the base.
+
+### Bug Fixes
+- Fixed issue: Test pane will not be populated properly in some cases #148.
+
+## [1.0.11] - 2024-07-11
+
+### Code Maintenance
+- Added tsc to run before npm run package, deprecating the direct use of vsce package.
+
+### Bug Fixes
+- Added type-checking for tests using tsc before running vitest to prevent type errors from blocking test execution.
+- Resolved a type error in src/editorDecorator.ts by adding missing properties to the testNode object.
+
+## [1.0.11] - 2024-07-23
+
+### Code Maintenance
+- Upgraded Prettier to version 3.3.3 and applied formatting on typescript files.
+- Applied xo on typescript files.
+- Added a prettier check to the GitHub Actions workflow.
+
+### Bug Fixes
+- Configured TypeScript to prevent emitting JavaScript files during TS tests.
+
+## [1.0.11] - 2024-07-30
+
+### Code Maintenance
+- Added tests in case VECTORCAST_DIR is not set, clicast isn't on PATH.
+- Enhanced testing possibilities by adding possible parameters for test groups.
+- Added tests for pythonUtilities.ts.
+- Updated the coverage upload process to send coverage-final.json to Codecov instead of clover.xml, ensuring aggregated coverage results are uploaded.
```

Comment:
"to send coverage-final.json to Codecov instead of clover.xml," can be removed, irrelevant detail